doc/source/tune-usage.rst

```diff
@@ -300,6 +295,17 @@ You often will want to compute a large object (e.g., training data, model weight
         }
     })
 
+Auto-Filled Results
+-------------------
+
+During training, Tune will automatically fill certain fields if not already provided. All of these can be used as stopping conditions or in the Scheduler/Search Algorithm specification.
```

Comment:
Btw we have some "magic" metrics that will show up on the status line, should those be mentioned here somewhere?